config/gregtech/worldgen/beneath/precious_vein.json

```diff
@@ -1,53 +1,53 @@
-{
-  "weight": 40,
-  "density": 0.3,
-  "max_height": 120,
-  "min_height": 50,
-  "dimension_filter": [
-    "is_nether"
-  ],
-  "surface_stone_material": "gold",
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [
-      14,
-      17
-    ]
-  },
-  "filler": {
-    "type": "ga_simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 100,
-          "value": "ore:silver"
-        },
-        {
-          "weight": 20,
-          "value": "ore:poor:precious_metal"
-        },
-        {
-          "weight": 50,
-          "value": "ore:precious_metal"
-        },
-        {
-          "weight": 10,
-          "value": "ore:rich:precious_metal"
-        },
-        {
-          "weight": 5,
-          "value": "ore:pure:precious_metal"
-        },
-        {
-          "weight": 10,
-          "value": "ore:poor:gold"
-        },
-        {
-          "weight": 5,
-          "value": "ore:gold"
-        }
-      ]
-    }
-  }
+{
+  "weight": 40,
+  "density": 0.3,
+  "max_height": 120,
+  "min_height": 50,
+  "dimension_filter": [
+    "dimension_id:10"

```

Comment:
I still think there should be some form of gold in the nether. One of the things that helped originally was the nether ores giving 2X the yeild. Maybe a similar (or lesser) change should happen in this vein to offset the loss?